src/test/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExtTest.kt

```diff
@@ -0,0 +1,43 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.FormBody
+import okhttp3.Request
+import org.junit.Test
+import kotlin.test.assertEquals
+
+/**
+ * @author doyaaaaaken
+ */
+class RequestExtTest {
+
+    @Test
+    fun `build curl command of simple get request`() {
+        val request = Request.Builder().url("https://postman-echo.com/get").build()
+        val actual = request.buildCurlCommand()
+
+        assertEquals("curl -X GET \"https://postman-echo.com/get\"", actual)
+    }
+
+    @Test
+    fun `build curl command of request with header`() {
+        val request = Request.Builder()
```

Comment:
Probably, we need to introduce `Request` dsl builder

Reply:
I cannot find `Request` dsl builder in existing code.
So, do you mean that we should implement new dsl method like this?
```
request {
            url = "https://postman-echo.com/post"
            post = requestBody {
                contentType = "application/x-www-form-urlencoded"
                body = ""
            }
}
```

Reply:
yeah, unfortunately, there is no one at the moment.
But I think it's better to introduce it as a separate issue later

Reply:
Yes, I agree!